WebApi/WebApiTest/ControllerTest/PostControllerTest.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using WebApi.Controllers;
+using WebApi.Models.POCO;
+using Microsoft.Extensions.Logging;
+using WebApi.Services;
+using WebApi.Services.Serives_Implementations;
+using WebApi.Services.Services_Interfaces;
+using WebApi.Database;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Moq;
+using WebApi.Models.DTO;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Linq;
+using NuGet.Frameworks;
+using Microsoft.VisualBasic;
+using WebApi.Database.Mapper;
+using Microsoft.AspNetCore.Mvc;
+using System.Linq.Expressions;
+using WebApi.Models.DTO.PostDTOs;
+
+namespace WebApiTest.ControllerTest
+{
+    public class PostControllerTest
+    {
+        int userID = 1;
+        DateTime in_time = DateTime.Now;
+
+        [Theory]
+        [InlineData(0, "Konrad Gaweda", 1, "RTV", "MyTitle", "Content", 41, false, false)]
+        [InlineData(1, "Jan Kowalski", 11, "RTV", "MyTitle1", "Conte nt1 ", 11, false, true)]
+        [InlineData(4, "Jan Gawęda", 1, "RTV", "MyTitle2", " Co ntent2 ", 33, true, false)]
+        [InlineData(7, "Konrad Kowalski", 32, "RTV", "MyTitle3", "Conten t3", 441, true, true)]
+        public void GetAll_Test(int in_id, string in_author, int in_authorID, string in_category,
+            string in_title, string in_content, int in_likesCount, bool in_isLiked, bool in_isPromoted)
```

Comment:
To samo co u Gosi. Unikamy używania DateTime.Now albo Today albo cokolwiek co będzie zmieniało swoją wartość. Wartość musi być stała i najlepiej jak przygotujesz ich kilka do testowania.